apis/telemetry/v1alpha1/shared_types.go

```diff
@@ -28,9 +29,12 @@ func (v *ValueFromSource) IsSecretKeyRef() bool {
 }
 
 type SecretKeyRef struct {
-	Name      string `json:"name,omitempty"`
+	// The name of the Secret containing the referenced value
+	Name string `json:"name,omitempty"`
+	// The name of the namespace containing the Secret with the referenced value.
```

Comment:
```suggestion
	// The name of the Namespace containing the Secret with the referenced value.
```
Namespace should be [capitalized](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#camelcase).